multires_image/cmake/export_components.cmake

```diff
@@ -0,0 +1 @@
+find_package(Boost REQUIRED COMPONENTS filesystem thread)
```

Comment:
Can you add a line ending after this to make the linter happy?

Reply:
I added a new line at the end of the file.